bitbot/links.go

```diff
@@ -20,16 +22,21 @@ var URLReaderTrigger = NamedTrigger{
 	Action: func(irc *hbot.Bot, m *hbot.Message) bool {
 		resp := lookupPageTitle(m.Content)
 		if resp != "" {
-			irc.Reply(m, lookupPageTitle(m.Content))
+			title := lookupPageTitle(m.Content)
 			if len(m.Content) > 70 {
-				irc.Reply(m, shortenURL(m.Content))
+				irc.Reply(m, shortenURL(m.Content, title))
+			} else {
+				irc.Reply(m, title)
 			}
 		}
 		return true
 	},
 }
 
-func shortenURL(uri string) string {
+func shortenURL(uri string, title string) string {
```

Comment:
Let's leave this function signature the same and extract the URI + concatenate the message elsewhere. The shortenURL function should work as advertised and only shorten URLs, not also extract and concatenate.